Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
Can you remove this chunk as this will break compilation on c8s? We have a fix for this in #521 but it's not merged yet.

Reply:
Sure. I wouldn’t normally hand-edit a `Cargo.lock` file, but it ought to be safe enough to do so in this case.

Reply:
Thanks! Agreed, that is not ideal and only a temporary solution. Hopefully the real fix will land soon.